mix.exs

```diff
@@ -46,14 +59,35 @@ defmodule SaladUI.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:phoenix_live_view, "~> 0.20.1"},
       {:tails, "~> 0.1.5"},
+      {:phoenix, "~> 1.7"},
+      {:heroicons, "~> 0.5.3"},
+      {:phoenix_ecto, "~> 4.4"},
+      {:phoenix_html, "~> 4.0"},
+      {:mix_test_watch, "~> 1.2"},
+      {:phoenix_live_view, "~> 0.20.1"},
+      {:phoenix_html_helpers, "~> 1.0"},
```

Comment:
What do you use phoenix_html_helpers for?

Reply:
We don't need it since we are using the latest phoenix with Phoenix.Components.